Docs for re-register, and node+delegator (#220)
onflow/flow: docs/content/staking/locked-delegation-guide.mdx

and transfer their locked tokens to the new account by requesting authorization from the token admin.

We plan on adding functionality in the future to allow accounts to delegate to multiple nodes.

## Stake from the Same Account

> An account who registers a delegator with locked tokens can also register a delegator in the same account. 

Reviewer: ```suggestion
An account that registers as a delegator with locked FLOW can also register a staked node in the same account.
```

> See the Locked Staking Guide for how to register a node.

Reviewer: ```suggestion
See the [Locked Staking Guide](../locked-staking-guide) for how to register a node.
```
